### PR TITLE
Updated feedstock with conda smithy

### PR DESCRIPTION
**Pull request by melund**
@ocefpaf

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,27 +6,28 @@ language: objective-c
 env:
   matrix:
     
-    - CONDA_PY=27
```
Comment (ocefpaf) [inline .travis.yml]: @melund as you can see conda-smithy saw the `skip: True  # [py2k]` and removed the Python 2 builds for you.

PS: So xonsh is Python 3 only now?

File: recipe/bld.bat [hunk 2]
```diff
@@ -1 +0,0 @@
-python setup.py install --single-version-externally-managed --record=record.txt
```
Comment (ocefpaf) [inline recipe/bld.bat]: :+1:

File: .travis.yml (hunk 1 above)
Comment (pelson) [inline .travis.yml]: Coor! That's clever :smile:

File: appveyor.yml [hunk 3]
```diff
@@ -42,18 +37,28 @@ platform:
...
     - cmd: python bootstrap-obvious-ci-and-miniconda.py %CONDA_INSTALL_LOCN% %TARGET_ARCH% %CONDA_PY:~0,1% --without-obvci
     - cmd: set PATH=%CONDA_INSTALL_LOCN%;%CONDA_INSTALL_LOCN%\scripts;%PATH%
     - cmd: set PYTHONUNBUFFERED=1
 
+    - cmd: conda config --set show_channel_urls true
     - cmd: conda install -c http://conda.binstar.org/pelson/channel/development --yes --quiet obvious-ci
     - cmd: conda config --add channels http://conda.binstar.org/conda-forge
     - cmd: conda info
     - cmd: conda install -n root --quiet --yes conda-build anaconda-client jinja2 setuptools
-    - cmd: if "%TARGET_ARCH%" == "x64" if "%CONDA_PY%" == "34" conda install conda-build=1.20.0 --yes
```
Comment (pelson) [inline appveyor.yml]: Careful. You want to keep this line until we have a conda-smithy fix, or until conda-build has been released with a fix.

File: .travis.yml (hunk 1 above)
Comment (ocefpaf) [inline .travis.yml]: @pelson I wonder who designed that :stuck_out_tongue_winking_eye:

Comment (melund) [inline .travis.yml]: @pelson Pretty neat, and yes Xonsh is Python 3 only

File: appveyor.yml (hunk 3 above)
Comment (melund) [inline appveyor.yml]: Ok. I am adding it back in...

**Comment by pelson** [comment]
You've gone green :+1:

**Comment by ocefpaf** [comment]
> You've gone green :+1: 

@melund I guess that what @pelson means is that it is OK to self-merge changes that brings only updates like this one.

**Pull request merged**